app/components/phase_banner.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="govuk-width-container">
-  <div class="govuk-phase-banner app-!-print-display-none">
+  <div class="govuk-phase-banner <%= @stacked ? 'app-phase-banner__stacked' : '' %> app-!-print-display-none">
```

Comment:
What does ‘stacked’ mean? Is it just that there's no border?

Reply:
That it's to be stacked on top of another element. Suggestion for a better name for this param v welcome!